src/template.ejs

```diff
@@ -13,6 +13,9 @@
     <% } for (let js of htmlWebpackPlugin.files.js) { %>
     <script src="<%= path.relative(dir, js) %>" defer></script>
     <% } %>
+    <link href="<%= htmlWebpackPlugin.options.icon %>" rel="shortcut icon" type="image/png"/>
+    <link href="chrome://browser/content/extension.css" rel="stylesheet">
```

Comment:
This should definitely be happening automatically, but just threw it in there for now to test

Reply:
I'm not actually sure it should be automatic, since we're just opening a page via the WebExtension. There's nowhere I know of to set `browser_style` the way we're doing things. It might make sense to have this be optional though (e.g. pass `browser_style` to the *template*).

Reply:
>I'm not actually sure it should be automatic, since we're just opening a page via the WebExtension

Ah yeah, its usually just for popups and options pages I think? MDN got me a little confused.

For my own reference again: https://dxr.mozilla.org/mozilla-central/rev/955840bfd3c20eb24dd5a01be27bdc55c489a285/browser/components/extensions/ext-utils.js#62

Needs more rethinking but low priority, will come back later.